05-Chile/02-R/Scripts/Data_work_explanation.Rmd

```diff
@@ -0,0 +1,366 @@
+
+The emergency portal can only partially help us to identify which tenders are covid-related since the merging rate with the main sample is only `r p_covid_match` \%.
+
+```{r fig-margin, echo=FALSE}
+
+knitr::include_graphics(file.path(github_dir, "auxilary_files", "Chile Compra.png"))
+
+```
+
+# Data Cleaning
+
+## Offer Value - *AMT\_VALUE\_AWARDED*
+
+> I replace all negative numbers as missing values. 
+
+I replace `r n_negative_amount_awarded` values as missing: they are negative numbers.
+
+> I replace irregular cases as missing. 
+
+`r n_case_check_1` observations present the following irregularity: quantity awarded is equal to total price offered (very weird) plus the value awarded is the product of these two terms (even more 
+weird). After checking a sample of 10 cases, I realize that the value awarded is equal to the price 
+offered. I replace them.
+  
+```{r check-url-1, echo = FALSE, results='asis', warning = FALSE, header=FALSE}
+
+
+check_1 <- check_1 %>% 
+  mutate(
+    ID_TENDER = paste0('<a href="', URL_TENDER, '">',ID_TENDER,'</a>')
+  ) %>% 
+  select(-URL_TENDER)
+
+datatable(check_1, escape=FALSE,
+  rownames = FALSE, extensions = c("Buttons",'Scroller'),
+  options = list(dom = 'Bfrtip',
+                                   buttons = c('csv','excel'),
+                                   scrollY = "500px",
+                                   scrollX = TRUE,
+                                   scroller = TRUE
+  )) 
+
+```
+
+> I replace the rest of irregular cases as missing. 
+
+`r n_case_check_2` observations present the following irregularity: AMT_QUANTITY_AWARDED, AMT_VALUE_AWARDED and AMT_TOTAL_PRICE are equal to 1. Then, I code as missings since I believe they are all mistypes.
+
+```{r check-url-2, echo = FALSE, results='asis', warning = FALSE, header=FALSE}
+
+check_2 <- check_2 %>% 
+  mutate(
+    ID_TENDER = paste0('<a href="', URL_TENDER, '">',ID_TENDER,'</a>')
+  ) %>% 
+  select(-URL_TENDER)
+
+datatable(check_2, escape=FALSE,
+  rownames = FALSE, extensions = c("Buttons",'Scroller'),
+  options = list(dom = 'Bfrtip',
+                                   buttons = c('csv','excel'),
+                                   scrollY = "500px",
+                                   scrollX = TRUE,
+                                   scroller = TRUE
+  )) 
+
+```
+
+> I convert all the values in USD following the instructions provided on the <a href="https://transparenciachc.blob.core.windows.net/oc-da/ParidadMoneda.csv">Chile Compra</a> platform. 
+
+> I trim the values at the 99th percentile.
+
+The following box-plots illustrate the variation of the value using different percentiles while trimming.  
+
+# Task 1 
+
+**Define variable: medical equipment** We can define items as medical equipment following the UNSPC code: item codes that start with "42" are considered as such. The following table shows the distribution of "Medical Equipment" among "Tender Emergency".
+
```

Comment:
@rdoino-wb can you follow the Leandro's method to classify the products? (I am not sure if you did) Thank you!